src/utils/drops.ts

```diff
@@ -0,0 +1,117 @@
+import BigNumber from "bignumber.js";
+
+import { ValidationError } from "../common/errors";
+
+const DROPS_PER_XRP = 1000000.0;
+const MAX_FRACTION_LENGTH = 6;
+const BASE_TEN = 10;
+const VALID_XRP = /^-?[0-9]*\.?[0-9]*$/u;
+const VALID_DROPS = /^-?[0-9]*\.?[0-9]*$/u;
```

Comment:
Why are we allowed to have decimals of a drop?

Reply:
I think those are just for preliminary checks. Replaced the constants w/ the RegEx